src/components/Validation.js

```diff
@@ -0,0 +1,18 @@
+import * as yup from 'yup';
+
+export const ValidationSchema = yup.object().shape({
+    email: yup
+      .string()
+      .matches(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+        'Email must have a number, special character, small and capital alphabets.',
+      )
+      .required('Email is required'),
+    password: yup
+      .string()
+      .matches(/\w*[a-z]\w*/, 'Password must have a small letter')
```

Comment:
add strings in separate string file.. add react-native-localization and use this library.. nothing must be hardcoded